### PR TITLE
master stability: counting only scheduled builds now

### PR DESCRIPTION
**Pull request by rzinurov**
@rasmusbergpalm

### CODE REVIEW

File: src/jenkins/ui/MasterStatus.js [hunk 1]
```diff
@@ -11,16 +11,24 @@ export default class MasterStatus extends React.Component {
...
             });
-            return onlyMaster;
+            var fullPackage = true;
+            _.forEach(parameters, (value, name) => {
+                if (name === "PACKAGE_PATH" && value !== "com/tradeshift") {
+                    fullPackage = false;
+                }
+            });
+            var username = build.getUserFullName();
+            return onlyMaster && fullPackage && username === null;
```
Comment (rasmusbergpalm) [inline src/jenkins/ui/MasterStatus.js]: Can't you just check that the upstream build is 'periodic-master-ITs' ??

Comment (rzinurov) [inline src/jenkins/ui/MasterStatus.js]: Let's take a look at that together during our IT Dashboard introduction today

**Comment by rzinurov** [comment]
@rasmusbergpalm the implementation has been updated according to our discussion today

**Comment by rasmusbergpalm** [comment]
Awesome! :baby:

**Pull request merged**